build.gradle

```diff
@@ -313,6 +313,12 @@ project(':core') {
     from sourceSets.test.output
   }
 
+  task publishDocs(type: Jar) {
```

Comment:
Do you mean to say that, docs tar file should be part of kafka distribution and inside kafka/site folder?

Current patch will upload kafka-docs jar file to maven and kafka distribution.